java/code/webapp/WEB-INF/pages/common/fragments/systems/system_list_fragment.jspf

```diff
@@ -14,6 +15,10 @@
   </c:otherwise>
 </c:choose>
 
+<c:if test="${current.rhnProxy}">
```

Comment:
Not sure if this works when `current.rhnProxy` is empty. Have you tested it?

Reply:
Yes, like the other comment, it was tested manually and the screenshot taken from it. The reason is because of the [DTO format](https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/frontend/dto/SystemOverview.java#L165) I didn't want to change

Reply:
ok